setup/setup-lokole.sh

```diff
@@ -644,7 +644,19 @@ main() {
   fi
 }
 
-main 2>&1 | tee --append "\${sync_logfile}"
+main_with_retry() {
+  attempts=0
+  timeout=5
+  until "$(attempts -ge 5)"
+  do
+    main 2>&1 | tee --append "\${sync_logfile}" && break
```

Comment:
To make this slightly more explicit, I'd recommend to use `if` to check for command success instead of `&&`:

```suggestion
    if main 2>&1 | tee --append "\${sync_logfile}"; then
      break
    fi
```

Not that this is required right now, but this syntax also makes it easier to add more commands to the "command successful" block.